pages/profile/[username].tsx

```diff
@@ -101,7 +116,11 @@ export default function Profile(props: PageProps) {
                     {data?.allow_anon_dm ? (
                         <>
                             <div className="flex flex-grow bg-gray05 rounded-xl py-1 px-2">
-                                <div contentEditable onInput={(e) => setMessage(e.currentTarget.textContent)} className="bg-transparent min-h-[24px] w-full" placeholder="Send a message..." />
+                                <textarea ref={textAreaRef} rows={1} onChange={(e) => {
+                                    setMessage(e.target.value)
+                                }} className="bg-transparent min-h-[24px] w-full"
+                                    style={{ resize: 'none' }}
+                                    placeholder="Send a message..." />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The textarea element does not have any form of input validation or sanitization. This can potentially lead to security issues such as Cross-Site Scripting (XSS) attacks if the user input is rendered directly somewhere else in the application. Consider adding some form of input validation or sanitization to prevent these types of vulnerabilities.


<!-- This is an auto-generated comment by OSS CodeRabbit -->